Makefile

```diff
@@ -24,15 +41,18 @@ $(CRATES):
 $(WORKSPACES_WITH_RUST_MODULES):
 	$(MAKE) -C $@ install
 
+.PHONY: toolchain
+toolchain: ## install toolchain
+	scripts/toolchain/install-toolchain.sh
+
 .PHONY: deps
-deps: ## install dependencies
-# only use corepack on non-nix systems
-	[ -n "${NIX_PATH}" ] || corepack enable
-	command -v rustup && rustup update || echo "No rustup installed, ignoring"
+deps: ## install dependencies in CI
```

Comment:
Removing the previous steps from deps makes the setup not work on systems which are unsupported by the install toolchain script. I don't think that is the right thing to do unless you want to expand that script to cover more setups right away. As is, this doesn't work on my machine.

Reply:
changed